src/shared/defaultAwsContext.ts

```diff
@@ -56,8 +60,16 @@ export class DefaultAwsContext implements AwsContext {
         return this.currentCredentials?.accountId
     }
 
-    public getCredentialDefaultRegion(): string | undefined {
-        return this.currentCredentials?.defaultRegion
+    public getCredentialDefaultRegion(): string {
+        const credId = this.currentCredentials?.defaultRegion ?? DEFAULT_REGION
```

Comment:
this is not correct. please review what i suggested in https://github.com/aws/aws-toolkit-vscode/pull/1844#discussion_r661026658

credId is only for use as a key in `logged`. We still need to return 

    this.currentCredentials?.defaultRegion ?? DEFAULT_REGION